.github/workflows/lint.yml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Lint all codez
+on:
+  push:
+    tags: 'release-*'
+    branches:
+      - dev
+      - release
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  go:
+    name: Lint go files
+    runs-on: ubuntu-latest
```

Comment:
This is currently `18.04` and they're about to roll out 20.04 on this tag. We should just specify `ubuntu-20.04` for all of them; moving targets tend to be a surprise at the least convenient times.

Reply:
good point. Done!